packages/eas-cli/src/submit/utils/builds.ts

```diff
@@ -8,14 +8,29 @@ export async function getRecentBuildsForSubmissionAsync(
   appId: string,
   { limit = 1 }: { limit?: number } = {}
 ): Promise<BuildFragment[]> {
-  return await BuildQuery.viewBuildsOnAppAsync(graphqlClient, {
+  const builds = await BuildQuery.viewBuildsOnAppAsync(graphqlClient, {
     appId,
     limit,
     offset: 0,
     filter: {
       platform,
       distribution: DistributionType.Store,
-      status: BuildStatus.Finished,
+      status: BuildStatus.InProgress,
```

Comment:
What about `NEW` and `IN_QUEUE` builds? I believe we should handle submitting these as well.

Reply:
I don't see why not, I'll add them